extensions/spark/kyuubi-spark-lineage/pom.xml

```diff
@@ -38,15 +38,22 @@
         </dependency>
 
         <dependency>
-            <groupId>commons-collections</groupId>
-            <artifactId>commons-collections</artifactId>
-            <scope>test</scope>
+            <groupId>org.apache.kyuubi</groupId>
```

Comment:
`kyuubi-events` and `kyuubi-common` dependencies are only used in `KyuubiEventDispatcher`, so we can still use lineage plugin without kyuubi